docs/docs/vulnerability/detection/language.md

```diff
@@ -43,4 +43,5 @@ Example: [Dockerfile](https://github.com/aquasecurity/trivy-ci-test/blob/main/Do
 [^9]: ✅ means "enabled" and `-` means "disabled" in the rootfs scanning
 [^10]: ✅ means "enabled" and `-` means "disabled" in the filesystem scanning
 [^11]: ✅ means "enabled" and `-` means "disabled" in the git repository scanning
-[^12]: To scan a filename other than the default filename(`conan.lock`) use [file-patterns](../examples/others.md#file-patterns)
\ No newline at end of file
+[^12]: Dependency location is only supported in `json` and [SBOM](../../sbom/index.md) formats. SBOM uses `startline == 1 and endline == 1` for unsupported file types
```

Comment:
We should explain what is the dependency location first.

Reply:
Done